records_mover/records/__init__.py

```diff
@@ -20,7 +21,9 @@
 from .records_types import RecordsFormatType, DelimitedVariant
 from .schema import RecordsSchema
 from .mover import move
-from .records_format import RecordsFormat, DelimitedRecordsFormat, ParquetRecordsFormat
+from .records_format import (
+    RecordsFormat, DelimitedRecordsFormat, ParquetRecordsFormat, AvroRecordsFormat
+)
```

Comment:
Let's make this a public interface similar to DelimitedRecordsFormat/ParquetRecordsFormat